tqdm.py

```diff
@@ -0,0 +1,50 @@
+import collections
+import time
+import copy
+
+
+class Tqdm(collections.Iterator):
```

Comment:
You should doc this class, as this is the main export of your package.

Reply:
Done